.rubocop.yml

```diff
@@ -51,6 +51,17 @@ Lint/HandleExceptions:
   StyleGuide: "#dont-hide-exceptions"
   Enabled: true
 
+# Supports --auto-correct
+Lint/InheritException:
+  Description: Avoid inheriting from the `Exception` class.
+  Enabled: true
+  EnforcedStyle: runtime_error
+  SupportedStyles:
+  - runtime_error
+  - standard_error
+  Exclude:
+  - truffleruby/src/main/ruby/core/exception.rb
```

Comment:
The value is as with any other cop, to catch a mistake which might be done by us or a contributor. We still have majority of the exceptions not a child of Exception which is what this cop reflects. The probability of a mistake is low I guess, but since this is not problematic for us (as mentioned we will run into this issue very rarely) then I see no point in discussing this cop, we can just stick with the default, which was what we have initially set to do.